katalog/istio-operator/jaeger-operator/kustomization.yaml

```diff
@@ -7,9 +7,13 @@ resources:
   - cluster_role.yaml
   - cluster_role_binding.yaml
   - ns.yml
+  - https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.22.1/deploy/crds/jaegertracing.io_jaegers_crd.yaml
```

Comment:
Are these files duplicated? By the way we merged #13 that removes remote manifests dependencies, i think we should remove all these remote links